include/inexor/vulkan-renderer/application.hpp

```diff
@@ -13,7 +13,8 @@
 
 namespace inexor::vulkan_renderer {
```

Comment:
This one later or do not add `inexor::vulkan_renderer` to that other.

Reply:
Done.